bin/metric-postgres-statsbgwriter.rb

```diff
@@ -91,13 +96,13 @@ def run
     ]
     con.exec(request.join(' ')) do |result|
       result.each do |row|
-        output "#{config[:scheme]}.bgwriter.checkpoints_timed", row['checkpoints_timed'], timestamp
-        output "#{config[:scheme]}.bgwriter.checkpoints_req", row['checkpoints_req'], timestamp
-        output "#{config[:scheme]}.bgwriter.buffers_checkpoint", row['buffers_checkpoint'], timestamp
-        output "#{config[:scheme]}.bgwriter.buffers_clean", row['buffers_clean'], timestamp
-        output "#{config[:scheme]}.bgwriter.maxwritten_clean", row['maxwritten_clean'], timestamp
-        output "#{config[:scheme]}.bgwriter.buffers_backend", row['buffers_backend'], timestamp
-        output "#{config[:scheme]}.bgwriter.buffers_alloc", row['buffers_alloc'], timestamp
+        output "#{config[:scheme]}.bgwriter.#{config[:database]}.checkpoints_timed", row['checkpoints_timed'], timestamp
```

Comment:
what if they don't specify a database name? I think we need to check if it is set and then change the output on it.